core/modules/query/BoolFactorTerm.h

```diff
@@ -73,6 +75,9 @@ class BoolFactorTerm {
     /// Get a vector of the ValueExprs this contains.
     virtual void findValueExprs(std::vector<std::shared_ptr<ValueExpr>>& vector) const = 0;
 
+    /// Get a vector of references to pointers to the ValueExprs this contains.
+    virtual void findValueExprRefs(ValueExprPtrRefVector& vector) = 0;
```

Comment:
no, because it's a vector of reference_wrapper (because the object that uses the items in the vector needs to be able to modify the contents of the found pointer), a const object (inside a const method) can't be assigned to the reference_wrapper.